.gitignore

```diff
@@ -4,6 +4,7 @@ public/upload.js
 public/download.js
 public/version.json
 public/l20n.min.js
+public/bundle.js
```

Comment:
just `rm public/bundle.js` from your local machine and this shouldn't be necessary